### DISCUSSION TITLE
F-Droid?

### DISCUSSION THREAD

**Discussion by olof-nord** [anyproto/contributors#36] [Anytype apps]
I like the look of this project!
Being an ipen-source project, I understand an F-Droid release of the Android app is possible. Is this on the road map? How do you reason around this topic?

**Comment by fuksman** [comment]
Hello @olof-nord, I've checked the requirements of F-Droid and it doesn't look possible to submit `anytype-kotlin` there. F-Droid accepts projects with OSI-approved licenses, and Anytype apps are licensed under a source-available license, which is not approved by OSI.

**Reply by Narvey** [reply]
However, anyone can make an F-Droid repo. I wonder if someone should make a source-available repo?

**Reply by akumar-xyz** [reply]
That's a good idea. 
An officially blessed f-droid release/repo will be super useful for people running de-googled devices
For reference: https://f-droid.org/en/docs/Setup_an_F-Droid_App_Repo/

izzyOnDroid might also be a good option but I am not sure if Anytype meets their criteria either:
https://gitlab.com/IzzyOnDroid/repo#how-can-i-get-my-favorite-app-listed-there
